ursula_cli/shell.py

```diff
@@ -527,6 +542,14 @@ def parse_args():
         '--heat-stack-name', default=None,
         help='Name of the heat stack when heat provisioner is used',
     )
+    parser.add_argument(
+        '--heat-stack-update', default=False, action='store_true',
+        help='Update the heat stack',
+    )
+    parser.add_argument(
+        '--heat-parameters', default=None,
+        help='Semicolon separated list of KEY=VALUE parameters for heat',
```

Comment:
I guess my issue with this is that it'll need to be escaped in the shell or quoted, or it's gonna break.  Anyway we can make this more intuitive or maybe shlexable?  If this is the _most_ ideal way to separate a list for heat, I get it, but this might cause some confusion.